### DISCUSSION TITLE
Lambda redirector help

### DISCUSSION THREAD

**Discussion by badLlama121** [BishopFox/sliver#1370] [Q&A]
I am playing around with using Lambda as a redirector, and having issues.  I have confirmed my function works, and i can hit it from a browser and make it kick.

When i add the function URL to generate a beacon, nothing happens.  The implant generates, I run it, but looking at both wireshark and proxy logs, I see no attempts to reach out to the function URL provided in the generate command.  Wondering if anyone else has seen similar issues?

**Comment by moloch--** [comment]
You may want to try `--debug` when you generate the implant.
